product_pricelist_direct_print_company_group/wizards/product_pricelist_print.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class ProductPricelistPrintCompanyGroup(models.TransientModel):
+    _inherit = "product.pricelist.print"
+
+    @api.model
+    def _get_sale_order_domain(self, partner):
+        domain = super()._get_sale_order_domain(partner)
+        # If any parter has company_group_id defined with the selected partner, it means
+        # that the partner selected is a company group.
+        partners_grouped = self.env["res.partner"].search(
```

Comment:
Now you can use the field added in:
https://github.com/OCA/partner-contact/pull/1239

I use it at here:
https://github.com/OCA/contract/pull/797/files#diff-4cbf45e260f90f16486df50dd7c169ad2918d55b0abc9603f86c9298643f544aR21